superset-frontend/spec/fixtures/mockDashboardLayout.js

```diff
@@ -189,6 +189,49 @@ export const dashboardLayoutWithTabs = {
   future: [],
 };
 
+export const dashboardLayoutWithTabsCombined = {
```

Comment:
Could we have a more descriptive name here: how is this functionally different from `dashboardLayoutWithTabs`?